Cargo.toml

```diff
@@ -23,13 +23,13 @@ bytes = "0.4.12"
 http = "0.1.17"
 
 [dev-dependencies]
-http-service-hyper = { path = "http-service-hyper", version = "0.2.0" }
+http-service-hyper = { path = "./http-service-hyper" }
 
 [dependencies.futures-preview]
 version = "0.3.0-alpha.16"
 
 [patch.crates-io]
```

Comment:
You might be able to remove this whole `patch` section, I guess it was a consequence of `http-service-lambda` not having a path dependency on `http-service`. (Although the `tide` patch may still be needed until an update of that is released).

Reply:
Oh, actually no, this is probably needed to force `tide` to use the local `http-service` instead of pulling an additional one from crates.io, this sort of circular example is sort of annoying.